app/controllers/users_controller.rb

```diff
@@ -311,7 +311,7 @@ def create
       return fail_with("login.reserved_username")
     end
 
-    if user = User.where(staged: true).find_by(email: params[:email].strip.downcase)
+    if user = User.where(staged: true).find_by_email(params[:email].strip.downcase)
```

Comment:
Sure, although do we want to keep `find_by_email` as a deprecated method so we don't suddenly break every plugin which makes use of it?

> Perhaps find_by_user_email?

The way the method is currently written, that would be the best name. Though, I'm now wondering if actually in all these places we want to use a `find_by_primary_email` method, and save `find_by_user_email` for any functionality we want to change as part of phase 2.